NumberValidator/Validators/IN/AADHAAR.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Linq;
+using NumberValidator.Helpers;
+using System.Text.RegularExpressions;
+
+namespace NumberValidator.Validators.AADHAAR
```

Comment:
namespace is not correct. It needs to be `NumberValidator.Validators.[two letter country code]`, nothing else. Then you start implementing your particular validation inside such namespace. You do that change and you will see the code readability increases.